runtests.sh

```diff
@@ -778,6 +870,12 @@ alltests() {
   echo ""
 }
 
+export_testvars
```

Comment:
These lines should not be in the commit, or?  It prevents people from getting into the interactive, menu-based testing mode where they can configure parameters for the tests and run different tests.